docs/source/python_api/exceptions/mlflow.exceptions.rst

```diff
@@ -0,0 +1,16 @@
+:orphan:
```

Comment:
This file was included to fix broken exception references throughout the docs (e.g., `MlflowException, `ExecutionException`). For Sphinx to render these references properly, the target exceptions must exist somewhere in the documentation. This new page documents relevant MLflow exceptions and inserts them under the `exceptions` directory, which is *not* included in the Table of Contents / Documentation index. This decision was made because modules in the index contain user-facing APIs that provide platform functionality, while `mlflow.exceptions` only defines exceptions surfaced by other API calls.